1.12.0/rockcraft.yaml

```
@@ -29,7 +29,7 @@ parts:
     build-packages:
       - build-essential
     build-snaps:
-      - go/1.21/stable
+      - go/1.22/stable
```

looks like [1.12.0](https://github.com/coredns/coredns/blob/v1.12.0/.go-version) wants 1.23 go version

```suggestion
      - go/1.23/stable
```